package/package.json

```diff
@@ -20,6 +20,5 @@
     ],
     "website": "https://docs.mesosphere.com/current/usage/service-guides/spark/",
     "selected": true,
-    "version": "$version",
-    "minDcosReleaseVersion": "1.7"
+    "version": "{{package-version}}"
```

Comment:
OK, because I'm probably not going to change the spark release job to use the release_builder.